### PR TITLE
build for sse3 by default to avoid avx2 instructions being picked up …

### PR DESCRIPTION
**Pull request by mboisson**
…by distutils

This PR goes with PR https://github.com/ComputeCanada/software-stack-custom/pull/13

### CODE REVIEW

File: build_wheel.sh [hunk 1]
```diff
@@ -366,7 +366,14 @@ if [[ -z "$EBROOTGENTOO" ]]; then
 	module --force purge
 	module load nixpkgs gcc/7.3.0
 else
-	module --force purge
```
Comment (bartoldeman) [inline build_wheel.sh:369]: seems a little strange to have this in both branches now?

Comment (bartoldeman) [reply build_wheel.sh:369]:
```
module --force purge
module load arch/${EBVERSIONARCH-sse3}
```
should work too

File: build_wheel.sh [hunk 2]
```diff
@@ -366,7 +366,14 @@ if [[ -z "$EBROOTGENTOO" ]]; then
...
+	if [[ -z "$EBVERSIONARCH" ]]; then
+		module --force purge
+		module load arch/sse3
+	else
+		ARCH_TO_LOAD="$EBVERSIONARCH"
+		module --force purge
+		module load arch/$ARCH_TO_LOAD
```
Comment (ccoulombe) [inline build_wheel.sh:375]: Why defining ARCH_TO_LOAD if it is not reused? EBVERSIONARCH would do :)

**Review by bartoldeman — CHANGES_REQUESTED** [review]

**Review by ccoulombe — APPROVED** [review]
Looks good otherwise

File: build_wheel.sh (hunk 1 above)
Comment (mboisson) [reply build_wheel.sh:369]: `$EBVERSIONARCH` won't be defined after `module --force purge`

Comment (bartoldeman) [reply build_wheel.sh:369]: ah yes!
```
ARCH_TO_LOAD="$EBVERSIONARCH"
module --force purge
module load arch/${EBVERSIONARCH:-sse3}
```
works then?

**Review by bartoldeman — CHANGES_REQUESTED** [review]

File: build_wheel.sh (hunk 2 above)
Comment (mboisson) [reply build_wheel.sh:375]: It is reused, 2 lines later.

File: build_wheel.sh (hunk 1 above)
Comment (mboisson) [reply build_wheel.sh:369]: fixed

**Review by ccoulombe — APPROVED** [review]

**Pull request merged**